### PR TITLE
DACT-1482-post-initial-ch01-filing

### PR DESCRIPTION
**Pull request by jditchburn**
https://companieshouse.atlassian.net/browse/DACT-1482

### CODE REVIEW

File: views/includes/directors.html [hunk 1]
```diff
@@ -36,11 +36,11 @@ <h2 class="govuk-summary-card__title">{{officerCard.officer.name}}</h2>
     </dl>
       <div class="govuk-!-padding-top-3">
         {% if updateEnabled === "true" %}
-        <button id="directorUpdateButton" name="update_director_details" class="govuk-button govuk-button--secondary govuk-!-margin-right-2" data-module="govuk-button" value="{{officerCard.appointmentId}}">
+        <button id="directorUpdateButton" name="updateAppointmentId" class="govuk-button govuk-button--secondary govuk-!-margin-right-2" data-module="govuk-button" value="{{officerCard.appointmentId}}">
```
Comment (olaych) [inline views/includes/directors.html:39]: I think there might be confusion here. I added this bit but removed it after speaking with Stokesy. The previous tickets that added the flags specified not to have this for corporate. Probably I got the information wrong but worth checking

**Review by cdrainCH — DISMISSED** [review]

**Review by jbestwickch — DISMISSED** [review]

File: src/controllers/remove.director.controller.ts [hunk 2]
```diff
@@ -85,7 +85,6 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 
     // Patch filing with etag and resignation date
     const updateFiling: OfficerFiling = {
-      referenceEtag: appointment.etag,
       resignedOn: resignationDate
```
Comment (PhilipGlover) [inline src/controllers/remove.director.controller.ts:88]: I thought the etag was needed for the termination, but if not.. ought the comment above this be amended?

Comment (jditchburn) [reply src/controllers/remove.director.controller.ts:88]: You're right it is still needed. The etag has been moved into the original POST when the 'remove director' button is first clicked to match the new CH01 post. But you're right I'll update the comment.

**Review by PhilipGlover — APPROVED** [review]
lgtm

**Pull request merged**